netdata/packaging: Documentation on distribution support matrix and functionality availability (#6552)
netdata/netdata: packaging/DISTRIBUTIONS.md

that we currently supporting and we are working on documenting our current state so that our users
have complete visibility over the range of support.

**Legend**:

> - **Version**: Operating System Version supported

Reviewer: ```suggestion
- **Version**: Operating system version supported
```

- **Family**: The family that the OS belongs to
- **CI Smoke Testing**: Smoke testing has been implemented on our CI, to prevent broken code reaching our users

> - **Ci Testing**: Testing has been implemented, to prevent broken or problematic code reaching our users

Reviewer: Capitalize the `I` in `CI`. Also, could we just say "bugs" instead of "broken or problematic code"?

```suggestion
- **CI Testing**: Testing has been implemented to prevent broken or problematic code reaching our users
```

- **CI Integration**: CI Integration has been fully implemented for the particular operating system version
- **CD**: Continious deployment support has been fully enabled for this operating system

> - **.DEB**: We provides a .DEB package for netdata, for that particular operating system

Reviewer: ```suggestion
- **.DEB**: We provide a `.DEB` package for that particular operating system
```

> - **.RPM**: We provides a .RPM package for netdata, for that particular operating system

Reviewer: ```suggestion
- **.RPM**: We provide a `.RPM` package for that particular operating system
```

- **Installer**: Running netdata from source, using our installer, is working for this operating system
- **Kickstart**: Kickstart installation is working fine for this operating system
- **Kickstart64**: Kickstart static64 installation is working fine for this operating system

> - **Community**: There is community support, of any kind for this operating system (Packaging maintainers, contributors, etc)

Reviewer: ```suggestion
- **Community**: This operating system receives community support, such as packaging maintainers, contributors, and so on
```
